bundles/org.eclipse.swt.tools/JNI Generation/org/eclipse/swt/tools/internal/NativesGenerator.java

```diff
@@ -77,9 +77,57 @@ public void generate(JNIClass clazz) {
 	sort(methods);
 	generateNativeMacro(clazz);
 	generateWarningSettings();
+	generateStaticMethods();
 	generateExcludes(methods);
 	generate(methods);
 }
+private void generateStaticMethods() {
+	if (!getOutputName().equals("gtk4")) {
+		return;
+	}
+
+	outputln("""
```

Comment:
This is probably not where these methods should go - but I don't know/see any existing place for such C side utility methods.

Please suggest better location?